MvvmCross/ViewModels/IMvxApplication.cs

```diff
@@ -12,8 +12,13 @@ public interface IMvxApplication : IMvxViewModelLocatorCollection
 
         void Initialize();
 
-        void Startup(object hint);
+        void Startup();
```

Comment:
I would leave this `void Startup(object hint);`, for one to not break it, and another, maybe people just don't want to use a generic. Also adjust the trace messages to do this.

Reply:
I agree.  I think this change was propose by @nmilcoff perhaps he has an explanation.

Reply:
Mhmm it wasn't me 🤔

Reply:
If someone wants to pass a parameter in (even if just object) they should use the generic. I don’t see much value in having the old method syntax as passing around object is just bad practice and should be avoid if possible.

Reply:
That's a fair point, but it's still a breaking change. IMO we should try to avoid breaking users code as much as possible, unless it's a major release. Otherwise we'll loose their trust

Reply:
I disagree. Break without reason I totally agree, but the point of minor release is that you can do minimal api changes ie no architectural changes. 
What we can’t afford to do is to stagnate and not clean up code for fear of breaking developers. It’s OSS and comes “as is”

Reply:
We also broke the MvxAppStart Startupmethod by changing the return parameter to an `object`.  So either we accept or refuse both breaking changes.